### PR TITLE
add vbo org create & delete endpoints

### PR DESCRIPTION
**Pull request by iamtash**
added methods to make requests to the following endpoints
- [Create a new O365 Organization](http://doc.10.api.iland.test/1.0/#/companies/createO365Organization)
- [Delete an O365 Organization](http://doc.10.api.iland.test/1.0/#/Office%20365%20Backup/removeOrganization)

### CODE REVIEW

File: src/sdk/model/company/company.ts [hunk 1]
```diff
@@ -614,6 +615,22 @@ export class Company extends Entity {
...
 
+  /**
+   * Create a new O365 Organization.
+   * @param {string} locationUuid
+   * @param {O365OrgCreateRequest} creationRequest
+   * @returns {Promise}
+   */
+  /* istanbul ignore next */
+  async createO365Organization(locationUuid: string, creationRequest: O365OrgCreateRequest): Promise<O365Organization> {
+    return Iland.getHttp().post(`companies/${this.uuid}/location/${locationUuid}/actions/create-o365-org`,
```
Comment (ariquinones) [inline src/sdk/model/company/company.ts:626]: Needs a forward slash here before companies `/`

File: src/sdk/model/company/company.ts [hunk 2]
```diff
@@ -614,6 +615,22 @@ export class Company extends Entity {
       creationRequest.json);
   }
 
+  /**
+   * Create a new O365 Organization.
+   * @param {string} locationUuid
+   * @param {O365OrgCreateRequest} creationRequest
+   * @returns {Promise}
```
Comment (ariquinones) [inline src/sdk/model/company/company.ts:622]: `{Promise<O365Organization>}`

File: src/sdk/model/company/__json__/o365-org-create-request-json.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+/* istanbul ignore next */
```
Comment (ariquinones) [inline src/sdk/model/company/__json__/o365-org-create-request-json.ts:1]: No need for the istanbul comment here, only applies to methods and classes. Need to just add normal docs for this interface

File: src/sdk/model/company/o365-org-create-request.ts [hunk 4]
```diff
@@ -0,0 +1,90 @@
+import { O365OrgCreateRequestJson } from './__json__/o365-org-create-request-json';
+
+/**
+ * O365 Organization creation request.
+ */
+/* istanbul ignore next */
+
```
Comment (ariquinones) [inline src/sdk/model/company/o365-org-create-request.ts:7]: Format. Shouldn't skip a line here

File: src/sdk/model/company/company.ts (hunk 1 above)
Comment (iamtash) [reply src/sdk/model/company/company.ts]: fixed!

File: src/sdk/model/company/company.ts (hunk 2 above)
Comment (iamtash) [reply src/sdk/model/company/company.ts]: fixed!

File: src/sdk/model/company/__json__/o365-org-create-request-json.ts (hunk 3 above)
Comment (iamtash) [reply src/sdk/model/company/__json__/o365-org-create-request-json.ts]: fixed!

File: src/sdk/model/company/o365-org-create-request.ts (hunk 4 above)
Comment (iamtash) [reply src/sdk/model/company/o365-org-create-request.ts]: removed!

**Pull request merged**